package.json

```diff
@@ -44,7 +44,12 @@
   },
   "homepage": "https://github.com/baselime/node-opentelemetry#readme",
   "peerDependencies": {
-    "@trpc/server": "^10.0.0"
+    "@trpc/server": "next"
+  },
+  "peerDependenciesMeta": {
+    "@trpc/server": {
+      "optional": true
+    }
```

Comment:
i assume you want this as not everyone will be using trpc and some package manager auto-installs peer dependencies unless they're marked optional